mlflow/legacy_databricks_cli/configure/provider.py

```diff
@@ -325,6 +325,12 @@ def _get_databricks_model_serving_config():
         OAUTH_CACHE_ENV_VAR = "DATABRICKS_DEPENDENCY_OAUTH_CACHE"
         OAUTH_CACHE_EXPIRATION_ENV_VAR = "DATABRICKS_DEPENDENCY_OAUTH_CACHE_EXIRY_TS"
         MODEL_SERVING_HOST_ENV_VAR = "DATABRICKS_MODEL_SERVING_HOST_URL"
+        DB_MODEL_SERVING_HOST_ENV_VAR = "DB_MODEL_SERVING_HOST_URL"
+
+        # read from DB_MODEL_SERVING_HOST_ENV_VAR if available otherwise MODEL_SERVING_HOST_ENV_VAR
+        host = os.environ.get(
+            DB_MODEL_SERVING_HOST_ENV_VAR, os.environ.get(MODEL_SERVING_HOST_ENV_VAR)
+        )
```

Comment:
```suggestion
        host = os.environ.get(DB_MODEL_SERVING_HOST_ENV_VAR) or os.environ.get(MODEL_SERVING_HOST_ENV_VAR)
```
minor nit for extensibility and readability